### ISSUE TITLE
Internal error in submit when vcs type is not set, but settings are provided for gerrit

### ISSUE THREAD

**Issue by k-dovgan** [Samsung/Universum#119]
Originally created on Tue, 28 Nov 2017 16:41:02 +0900

Steps to reproduce:
1. Set environment variables for both gerrit and p4:
   * GIT_REFSPEC
   * GIT_REPO
   * P4PASSWD
   * P4PORT
   * P4USER
2. Launch submit without setting `vcs-type`:
```
universum submit  --project-root <path> --commit-message "<msg>" --reconcile-list <full_path>
```
Expected result: error is reported regarding misconfiguration or missing parameter
Actual result: portion of internal exception is displayed (`cannot concatenate 'str' and 'NoneType' objects`)

**Comment by i-keliukh** [comment]
@k-dovgan, it looks like the bug is fixed already. Please find what commit fixed the bug and close it.

**Comment by i-keliukh** [comment]
My bad, the bug is not fixed. Below are logs from recent reproduce:

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/_universum/lib/utils.py", line 178, in excepted_function
    func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/_universum/lib/utils.py", line 201, in function_in_block
    return self.structure.run_in_block(func, block_name, pass_errors, self, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/_universum/modules/structure_handler.py", line 131, in run_in_block
    result = operation(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/_universum/submit.py", line 52, in execute
    edit_only=self.settings.edit_only)
  File "/usr/local/lib/python2.7/dist-packages/_universum/lib/utils.py", line 104, in function_to_run
    result = function(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/_universum/modules/vcs/perforce_vcs.py", line 139, in submit_new_change
    raise CriticalCiException("Workspace '" + self.settings.client + "' doesn't exist!")
TypeError: cannot concatenate 'str' and 'NoneType' objects
```

**Comment by i-keliukh** [comment]
Closed by #300.
